File: 1-Setup & Tooling/APIs & Keys/apicall.py
import os
import Anthropic

client = anthropic.Anthropic()
model=os.environ.get("LLM_MODEL", "claude-sonnet-5")
response = client.messages.create(
    model=model,
    max_tokens=256,
    messages=[{"role": "user", "content": "What is a neural network in one sentence?"}]
)
print(response.content[0].text)

File: 1-Setup & Tooling/APIs & Keys/apicall.ts
import Anthropic from "@anthropic-ai/sdk";

const client = new Anthropic();

const MODEL = process.env.LLM_MODEL ?? "claude-sonnet-5";

const response = await client.messages.create({
  model: MODEL,
  max_tokens: 256,
  messages: [{ role: "user", content: "What is a neural network in one sentence?" }],
});

console.log(response.content[0].text);